packages/loaders/json-schema/src/getComposerFromJSONSchema.ts

```diff
@@ -252,7 +252,7 @@ export function getComposerFromJSONSchema(
       if (subSchema.enum && subSchema.type !== 'boolean') {
         const values: Record<string, EnumTypeComposerValueConfigDefinition> = {};
         for (const value of subSchema.enum) {
-          let enumKey = sanitizeNameForGraphQL(value.toString());
+          let enumKey = sanitizeNameForGraphQL(value?.toString() ?? 'NULL');
```

Comment:
Instead of an enum value representing `null`, we can make that field `nullable` which is a better approach for GraphQL.